tests/test.c

```diff
@@ -506,13 +523,21 @@ int test_modules_execution()
 
 int main(int argc, char **argv)
 {
-    UNUSED(argc)
+    char **skip = NULL;
+    int len = 0;
+    for (int i = 0; i < argc;  ++i) {
+        if (strcmp(argv[i], "--skip") == 0) {
```

Comment:
Actually we may want to support --skip module1,module2,module3 instead so we can have other options at the same time. For example --beam to run tests with Beam instead of AtomVM.

Reply:
removed